app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  def update_commment_count
+    posts.comments_counter += 1
+  end
```

Comment:
## Optional
- [ ] I highly suggest that you use [Rails active callback](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the #update_comment_count method every time a new comment has been saved to the database, then make the update_comment_count method private